src/installer/installer.go

```diff
@@ -97,6 +103,15 @@ func (i *installer) InstallNode() error {
 	return nil
 }
 
+func (i *installer) getRchosImage() (string, error) {
```

Comment:
Let's have a similar function for the MCO.
I think the map should look like this:
```
{4.4: {"rhcos": "https://blabla",
        "mco": "mco-image"},
4.5: ...
}
```